README.md

```diff
@@ -70,6 +70,49 @@ sudo apt-get install ffmpeg
 npm install
 ```
 
+
+# Installation on Arch Based command line (worked for one machine)
```

Comment:
Follow the same lint for headers as above in the README.

Line 39:
```md
### Installation in Debian Based command line
```
It should be ###.